extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php

```diff
@@ -15,10 +15,17 @@ public static function isEnabled() {
 			return false;
 		}
 
-		$wgGlobalEnableSourcePoint = WikiFactory::getVarValueByName( 'wgGlobalEnableSourcePoint', Wikia::COMMUNITY_WIKI_ID );
-
-		return !empty( $wgEnableUsingSourcePointProxyForCSS ) || !empty( $wgGlobalEnableSourcePoint );
+		return !empty( $wgEnableUsingSourcePointProxyForCSS );
+	}
+	
+	public static function getSourcePointBootStrapCode() {
+		if ( !static::isEnabled() ) {
+			return PHP_EOL . '<!-- SourcePoint recovery disabled. -->' . PHP_EOL;
+		} 
+		$sourcePointScript = F::app()->sendRequest( 'ARecoveryEngineApiController', 'getBootstrap' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$sourcePointScript". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)